src/app/ubs-admin/components/ubs-client-bonuses/ubs-client-bonuses.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="bonuses-container">
+  <div class="no-bonuses-wrapper" *ngIf="!totalBonuses && !isLoading">
+    <span class="no-bonuses-text">Вам ще не нараховано бонусів.</span>
+    <a href="#" class="more-about-bonuses-link">Дізнатися за що нараховуються бонуси</a>
+  </div>
+
+  <div class="bonuses-exist-wrapper" *ngIf="totalBonuses && !isLoading">
```

Comment:
add general ng-container to have 1  ngIf with same check

Reply:
done